circuit/src/execution-delivery/src/message_assembly/substrate_gateway_protocol.rs

```diff
@@ -954,11 +970,12 @@ mod tests {
             assert_signed_payload(
                 actual,
                 submitter,
-                vec![to.encode(), value.encode(), vec![]],
+                vec![to, value, vec![]],
                 vec![GatewayExpectedOutput::Events {
                     signatures: vec![b"EscrowedTransfer(address,address,value)".to_vec()],
                 }],
-                vec![1, 0, 12, 8, 4, 6, 8, 4, 1, 0],
+                // vec![1, 0, 12, 8, 4, 6, 8, 4, 1, 0],
```

Comment:
@zannis different encoded call yields with different output signature

Reply:
Resolved in 5fe1fb5.